pyproject.toml

```diff
@@ -1,19 +1,46 @@
-[tool.black]
-line-length = 79
-target-version = ["py37", "py38"]
+[tool.poetry]
+name = "ledfx"
+version = "0.1.0"
+description = "The best thing since sliced germanium!"
+authors = ["Your Name <you@example.com>"]
+license = "MIT"
+readme = "README.md"
 
-[tool.isort]
```

Comment:
Can you add this back in, since otherwise black reformats everything - the number of files changed due to different formatting is pretty big.

Reply:
Good call, done.